### ISSUE TITLE
Test failures when running tests locally.

### ISSUE THREAD

**Issue by fermadeiral** [SpoonLabs/sorald#138]
The following two tests are failing in my computer (but not in Travis):

```
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   FirstFitSegmentationAlgorithmTest.segmentationTest:45 expected:<DummyT[wo].java> but was:<DummyT[hree].java>
[ERROR]   SoraldTreeBuilderAlgorithmTest.treeBuildTest:36 expected:<DummyT[wo].java> but was:<DummyT[hree].java>
[INFO] 
[ERROR] Tests run: 31, Failures: 2, Errors: 0, Skipped: 0
[INFO] 
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
```

@henry-lp, because of that, I should not deploy. Could you please try to run `mvn clean test` in your computer to see if you also have test failures?

**Comment by henry-lp** [comment]
Hi Fernanda, the test passed locally at my machine, but I think somehow the file structure order in a folder is read differently at your so I made the test independent of the order in [this PR](https://github.com/SpoonLabs/sorald/pull/139)

**Comment by fermadeiral** [comment]
Fixed via #139
